File: features/article/article/Outbox.go
package article

import (
	"github.com/hailongz/golang/db"
)

type Outbox struct {
	db.Object
	Uid	int64	`json:"uid" name:"uid" title:"发布者ID"`
	Mid	int64	`json:"mid" name:"mid" title:"动态ID"`
	Body	string	`json:"body" name:"body" title:"内容" length:"-1"`
	Options	interface{}	`json:"options,omitempty" name:"options" title:"其他数据" length:"-1"`
	Ctime	int64	`json:"ctime" name:"ctime" title:"创建时间"`
}

func (O *Outbox) GetName() string {
	return "outbox"
}

func (O *Outbox) GetTitle() string {
	return "发布的动态"
}

File: features/article/ts/Outbox.ts
import { int64 } from "./lib/less";

/**
 * 发布的动态
 * @type db
 */
export class Outbox {

    /**
     * ID
     */
    id: int64 = 0

    /**
     * 发布者ID
     * @index
     */
    uid: int64 = 0

    /**
     * 动态ID
     * @index
     */
    mid: int64 = 0

    /**
     * 内容
     * @length -1
     */
    body: string = ""

    /**
     * 其他数据
     * @length -1
     */
    options: any

    /**
     * 创建时间
     */
    ctime: int64 = 0
    
}
